composer.json

```diff
@@ -29,7 +29,7 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.5",
         "doctrine/persistence": "^2.0",
-        "sonata-project/admin-bundle": "4.x-dev",
+        "sonata-project/admin-bundle": "4.x-dev || ^4.0",
```

Comment:
I think we can leave just ^4.0

Reply:
It during the beta, it will allow us to fix the BC-break on this bundle even if it's not release as a new alpha.

Reply:
yeah I would also say so